src/app/content/highlights/selectors.ts

```diff
@@ -91,6 +95,11 @@ export const highlightLocationFiltersWithContent = createSelector(
   (locationFilters, totalCounts) => getHighlightLocationFiltersWithContent(locationFilters, totalCounts)
 );
 
+export const highlightColorFiltersWithContent = createSelector(
+  highlightLocationFiltersWithContent,
```

Comment:
highlightColorFiltersWithContent should be based on totalCounts, so that they don't change when location filters are changed